src/declaration.h

```diff
@@ -534,6 +535,7 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
+    Ojbc_FuncDeclaration objc;
```

Comment:
@yebblies - how ready is it for merging?  I can raise a PR now to kick off the review process.